src/modules/components/banners/notification/notification.demo.html

```diff
@@ -4,15 +4,137 @@
 
   <gov-tab heading="HTML Example">
 
-    <docs-example language="markup">
-      <div class="gov-notification success">
-        <i class="gov-icon"></i>
-        <div class="gov-notification-body">
-          Accepted case management information
+    <section aria-labelledby="notifications">
```

Comment:
Use the docs-example directive instead as it will automatically display the code an the example. Later we'll evaluate if we need to change the markup of the directive to make the change affect all the code examples.